package.json

```diff
@@ -21,10 +21,10 @@
   },
   "scripts": {
     "build": "npm run build:readme && npm run build:js",
-    "prebuild:js": "mkdir -p dist",
-    "build:js": "browserify lib/http.js --outfile dist/chai-http.js --standalone chaiHttp",
+    "prebuild:js": "rm -rf dist",
+    "build:js": "simplifyify lib/http.js --outfile dist/chai-http.js --bundle --minify --debug --standalone chaiHttp",
     "build:readme": "rm -rf README.md && node ./support/readme",
-    "start": "npm run build && http-server -c-1",
+    "start": "npm run build -- --watch & http-server -o -c-1",
```

Comment:
The build process is being run with `--watch`, so it'll be a long-running process, hence the need to run it in the background (via the single `&` there).  I can switch it to use `npm-run-all --parallel` instead.  AFAIK, that's essentially the same thing, except that it also works on Windows.  Is that correct?